src/openforms/submissions/tests/test_exports.py

```diff
@@ -255,3 +255,14 @@ def test_submission_export_with_mixed_fields(self):
                 "sub1.input2",
             ),
         )
+
+    @tag("gh-2389")
+    @freeze_time()
+    def test_submissions_of_forms_with_translation_enabled_have_language_codes(self):
+        SubmissionFactory.create(
+            form=FormFactory.create(translation_enabled=True),
+            language_code="en",
+        )
```

Comment:
```suggestion
        SubmissionFactory.create(
            form__translation_enabled=True,
            language_code="en",
        )
```
the beauty of factory boy is that you can chain the factory kwargs :)

Reply:
TIL